### PR TITLE
[PPP-4223] - Use of vulnerable component spring-core before 4.3.14, s…

### PR DESCRIPTION
**Pull request by ricardosilva88**
**DO NOT MERGE WITHOUT THE REST OF PRs:**
[pentaho/maven-parent-poms#95 (comment)](https://github.com/pentaho/maven-parent-poms/pull/95#issue-250486125)

### CODE REVIEW

File: pentaho-karaf-assembly/src/main/filtered-resources/etc-server/custom.properties [hunk 1]
```diff
@@ -94,8 +94,8 @@ org.osgi.framework.system.packages.extra= \
  org.pentaho.vfs.ui, \
  org.pentaho.xul.swt.tab, \
  org.slf4j.*; version\="1.7.7", \
- org.springframework.dao; version\="4.3.2.RELEASE", \
- org.springframework.security.*; version\="4.1.5.RELEASE", \
+ org.springframework.dao; version\="4.3.19.RELEASE", \
```
Comment (graimundo) [inline pentaho-karaf-assembly/src/main/filtered-resources/etc-server/custom.properties]: Please use maven filtering to fill in the version value.

File: pentaho-karaf-assembly/src/main/filtered-resources/etc/custom.properties [hunk 2]
```diff
@@ -109,8 +109,8 @@ org.apache.commons.logging.*; version\="1.1.3", \
  org.pentaho.xul.swt.tab, \
  org.pentaho.platform.*, \
  org.slf4j.*; version\="1.7.7", \
- org.springframework.dao; version\="4.3.2.RELEASE", \
- org.springframework.security.*; version\="4.1.5.RELEASE", \
+ org.springframework.dao; version\="4.3.19.RELEASE", \
+ org.springframework.security.*; version\="4.2.9.RELEASE", \
```
Comment (graimundo) [inline pentaho-karaf-assembly/src/main/filtered-resources/etc/custom.properties]: Please use maven filtering to fill in the version value.

File: pentaho-karaf-assembly/src/main/filtered-resources/etc-server/custom.properties (hunk 1 above)
Comment (ricardosilva88) [reply pentaho-karaf-assembly/src/main/filtered-resources/etc-server/custom.properties]: Agree, working on it

File: pentaho-karaf-assembly/src/main/filtered-resources/etc/custom.properties (hunk 2 above)
Comment (ricardosilva88) [reply pentaho-karaf-assembly/src/main/filtered-resources/etc/custom.properties]: Agree, working on it

**Pull request merged**